src/components/Application/ProgressBarNEW.tsx

```diff
@@ -82,14 +109,16 @@ const ProgressBarNEW: React.FC<ProgressBarProps> = ({ structure, requestRevalida
       )
     return (
       <Icon
-        name={valid ? 'check circle' : 'exclamation circle'}
-        color={valid ? 'green' : 'red'}
+        name={isStrictlylInvalid ? 'exclamation circle' : 'check circle'}
+        color={isStrictlylInvalid ? 'red' : 'green'}
         size={step ? 'large' : 'small'}
       />
     )
   }
 
   const sectionsList = Object.values(sections).map(({ details, progress, pages }, index) => {
+    const isIsStrictSection = !!strictSectionPage && strictSectionPage.sectionCode === details.code
```

Comment:
```suggestion
    const isStrictSection = !!strictSectionPage && strictSectionPage.sectionCode === details.code
```

Was the `!!` intentional? What is it for?

Reply:
to turn it from null to false, otherwise type checking fails further down

Reply:
I did one global replace to make sure we capture all of the places this was mistyped: c7093b4